dist/common/scripts/scylla_setup

```diff
@@ -417,4 +423,4 @@ if __name__ == '__main__':
             print(' SELinux.')
 
         if dist_name() == 'Ubuntu':
-            run('apt-get install -y hugepages')
+            run('apt-get install -y hugepages')
```

Comment:
This is the most obvious example - something is not right with your editor/laptop/OS... ;)

Reply:
Probably a tab instead of white spaces. Need to fix that too.

Reply:
This is because I remove the empty line at the end of the files I changed. I'll roll back these changes.